src/ui/profile_preferences.ui

```diff
@@ -680,9 +680,6 @@
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
-            <property name="fontFilters">
```

Comment:
I have experimented a couple of years ago with applying/removing this filter with a checkbox `Allow proportional fonts` and making it a per-profile option. {I think this was when I first tried to do GUI language switching} - I also put in a combo control to select the *script*  which might help some users to choose fonts that cover *their* needs:
> ### writingSystem : [QFontDatabase::WritingSystem](https://doc.qt.io/qt-5/qfontdatabase.html#WritingSystem-enum)
> This property holds the writing system that serves as a filter for the combobox
> If *script* is `QFontDatabase::Any` (the default), all fonts are listed.

Access functions:|&nbsp;
-- | --
QFontDatabase::WritingSystem | **writingSystem**() const
void | **setWritingSystem**(*QFontDatabase::WritingSystem*)

It was necessary to refresh the QFontCombo control if either setting was changed - but it was also necessary to remember the user's prior choices so that a *previously* selected font was available to be selected when the dialogue was reopened a second time...

Reply:
I think it's safe enough to try this simpler setup.

Reply:
The script option was quite revealing when choosing some non-Latin scripts IIRC - for instance when some Far Eastern ones (e.g. `QFontDatabase::TraditionalChinese`) were chosen it reduced the list of suitable fonts on the system I was using to only one or two - which was actually quite helpful given that I had tried to equip the system with a wide a choice as I could. With the ethnic origins I suspect for yourself @vadi2 I wonder whether you might want to find fonts which support `QFontDatabase::Cyrillic` on occasions...! :grin:

Reply:
I'd rather keep things simpler than adding more knobs and fobs, you know me :)